docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       POSTGRES_DB: ${DATABASE_NAME:-codecarbon_db}
       POSTGRES_PASSWORD: ${DATABASE_PASS:-supersecret}
       POSTGRES_USER: ${DATABASE_USER:-codecarbon-user}
-    image: postgres:12
+    image: postgres:13
```

Comment:
we can go to latest (no major feature upgrade impacts our schema), maybe in a separate PR

Reply:
Thanks, I've set it to 16